src/test/java/com/znsio/templates/allAlbumsResponse.json

```diff
@@ -0,0 +1,502 @@
+[
```

Comment:
Why we are adding complete response here as data?

Reply:
to common for all response of posts .. 
ok i will add only the concern response